docs/event-listener.md

```diff
@@ -0,0 +1,54 @@
+---
+title: "Gravitino event listener"
+slug: /event-listener
+date: 2024-04-18
+keyword: event listener
+license: Copyright 2024 Datastrato Pvt Ltd. This software is licensed under the Apache License version 2.
+---
+
+## Introduction
+
+Gravitino provides event listener mechanism to allow use to capture the event provides by Gravitino server to inject some custom operations.
+
+## Event 
+
+Gravitino generates an event following the completion of CREATE, DROP, PURGE, ALTER, UPDATE, LOAD, LIST operations. The events can be hooked to the following resources:
```

Comment:
What's the difference between `ALTER` and `UPDATE`?

Reply:
done